bin/check_conda_package_versions.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+"""Script to check the current cycle versions of TSSW software against the
+conda repository.
+
+Attributes
+----------
+CYCLE_REPO : `str`
+    The name of the cycle build repository.
+ENV_FILE : `str`
+    The file containing the cycle versions.
+
+Notes
+-----
+This script requires conda to be installed.
+"""
+
+import argparse
+import pathlib
+from subprocess import run
+
+import json
+
+CYCLE_REPO = "ts_cycle_build"
+ENV_FILE = "cycle/cycle.env"
+
+
+def main(opts):
+    """Read the CYCLE env file and look up the specified TSSW packages and
+    versions in the conda repository. Finally print a list of the packages and
+    the versions that could not be found.
+
+    Parameters
+    ----------
+    opts : `argparse.Namespace`
+        The script command-line arguments and options.
+    """
+    sal_version = "0.0"
+    xml_version = "0.0"
+    packages_not_found = {}
+    packages_to_skip = [
+        "ts-xml",
+        "ts-sal",
+        "ts-idl-git",
+        "ts-dds-community",
+        "ts-dds-community-conda-build",
+        "ts-dds-private",
+        "ts-dds-private-conda-build",
+        "ts-ATMCSSimulator",
+        "ts-scriptqueue",
```

Comment:
There is a conda package for the CSC part. Is that not being built any more?

Reply:
It turns out that the build failed so I am rebuilding it again now. I'll remove this one from the skip list.